### ISSUE TITLE
Makedb questions

### ISSUE THREAD

**Issue by dazhudou1122** [bbuchfink/diamond#126]
Hi Benjamin, 

I am trying to do some analysis for my labmate where I need to create a custom diamond database to blast. So I essentially merged the two columns in an excel file using this command: 
xargs -n1 < file > newfile
and I got the txt file where I used for the database. Below is a section of the txt file: 

>WP_004030875.1	MKLAILGAGCYRTHAASGITNFSRACEVAEQVGKPEIAMTHSTIAMGAELKELAGIDEIVVSDPVFDNDFTVIDDFEYEAVIEAHKKDPESIMPQIREKVNAVAKDLPKPPKGAIHFTHPEDLGFEVTTDDNEAVQDADWVMTWFPKGDMQMGIIKEFADNLKEGAILTHACTVPTTMFQKIFEDLSSDEMNIAPKVNVSSYHPGAVPEMKGQVYIAEGYASEDAICKLVDWGVAARGDAFKLPAELLGPVCDMCSALTAITYAGILSYRDSVMNILGAPAGFAQMMAKESLTQVTDLMNSVGIDHMEEKLDPGALLGTADSMNFGAAADVLPSVLEVLENRKGKGPTCNI

When I used diamond makedb it gives me this error: 
#CPU threads: 4
Database file: Merge_HydDB_Database_Information.txt
Opening the database file...  [0.000769s]
Error: Error reading input stream at line 2: Invalid character (ASCII 13) in sequence

Can you advice on that? Thanks:) 

Best,

Wenhan

**Comment by dazhudou1122** [comment]
There is a ">" in front of WP_004030875.1 but it did not show in the post.

**Comment by bbuchfink** [comment]
Hi Wenhan,

there seem to be some invalid carriage return characters in your file. Try
running this to remove them:

tr -d '\15\32' < inputfile > outputfile

Best regards,

Benjamin

**Comment by dazhudou1122** [comment]
Hi Benjamin, 

Thanks for the advice! I did what you suggested, and it seems that I don't have a tab between the last letter of the sequence and the ">" symbol: 
>WP_004030875.1
MKLAILGAGCYRTHAASGITNFSRACEVAEQVGKPEIAMTHSTIAMGAELKELAGIDEIVVSDPVFDNDFTVIDDFEYEAVIEAHKKDPESIMPQIREKVNAVAKDLPKPPKGAIHFTHPEDLGFEVTTDDNEAVQDADWVMTWFPKGDMQMGIIKEFADNLKEGAILTHACTVPTTMFQKIFEDLSSDEMNIAPKVNVSSYHPGAVPEMKGQVYIAEGYASEDAICKLVDWGVAARGDAFKLPAELLGPVCDMCSALTAITYAGILSYRDSVMNILGAPAGFAQMMAKESLTQVTDLMNSVGIDHMEEKLDPGALLGTADSMNFGAAADVLPSVLEVLENRKGKGPTCNI>WP_012955328.1
MKVAILGAGCYRTHAASGITNFSRACEVADATGKENISMTHSTIEMGAELLELAGVDEVVVADPVFDGEFTVVEDFDYAEVIAAHKAGNPEDVMPAIRAKVGELAETVPKPANGAIHFTHPEDLGMKCTTDDREAVADADWIMTWLPEGGMQPAIIEKFADVIKDGAIVTSACTIPTPGLNQIFEDLGKNVNVASYHPGAVPEMKGQVYIAEGFADQAAIDTLKDLGAKARGSAFTLPANMVGPVCDMCSAVTAITYAGLLSYRDTVTQILGAPAGFAQMMANEALTNVTKLMADEGIDKMDDALNPGALLGTADSMNFGPLSEIVPTILESLEKRSK

And diamond is giving me the Error: Invalid character ( ) in sequence error. I tried to search and replace ">" with " >" but that did not work. 

Best, 

Wenhan
